site/common/layout/code-show.vue

```diff
@@ -57,7 +56,7 @@ export default {
   &-content {
     position: relative;
     border-top: 1px solid #dfdfdf;
-    padding: 16px 12px;
+    padding: 17px 16px 15px 20px;
 
     &-title {
       position: absolute;
```

Comment:
77: could add margin-right to fix problem about border display
64: number with no unit